web/templates/settings/custom_user_profile_field.hbs

```diff
@@ -4,28 +4,28 @@
     <div class="field_hint">{{ field.hint }}</div>
     <div class="field">
         {{#if is_long_text_field}}
-        <textarea maxlength="500" class="custom_user_field_value settings_textarea">{{ field_value.value }}</textarea>
+        <textarea maxlength="500" class="custom_user_field_value settings_textarea" name="{{ field.name }}">{{ field_value.value }}</textarea>
```

Comment:
One minor issue with this is that the names of the elements are being set in proper case and not lower case, causing the `dialog_widget.get_current_values` function to return inconsistent values where some properties are in lower case and others are in proper case.

We can create a `Handlebars.registerHelper` to convert names in lowercase if necessary.

Reply:
Ok, yeah I agree would be weird to have the names inconsistent but I am not sure. We can leave it as it is for now and update them later if needed after feedback from Tim. But I guess it works as expected now without any changes made to the case and the button is enabled and disabled correctly.

Reply:
I feel like we would be better served refactoring this code path to use the `field.id` (or is it `.field_id`?) value as a unique identifier for a field, rather than a string.